python/repeated.c

```diff
@@ -187,9 +187,29 @@ PyObject* PyUpb_RepeatedContainer_GetOrCreateWrapper(upb_array* arr,
   return ret;
 }
 
+static PyObject* PyUpb_RepeatedContainer_MergeFrom(PyObject* _self,
+                                                   PyObject* args);
+
+PyObject* PyUpb_RepeatedContainer_DeepCopy(PyObject* _self, PyObject* value) {
+  PyUpb_RepeatedContainer* self = (PyUpb_RepeatedContainer*)_self;
+  PyUpb_RepeatedContainer* clone = (void*)PyType_GenericAlloc(Py_TYPE(_self), 0);
+  const upb_fielddef* f = PyUpb_RepeatedContainer_GetField(self);
+  if (clone == NULL) return NULL;
```

Comment:
might as well move this to immediately after the alloc call

Reply:
Done.